### PR TITLE
Adding i18n Greek language

### PR DESCRIPTION
**Pull request by PeteShag**
Included an  i18n translation of JHipster to Greek.

### CODE REVIEW

File: generators/generator-base.js [hunk 1]
```diff
@@ -292,6 +292,7 @@ Generator.prototype.getAllSupportedLanguageOptions = function () {
         {name: 'French', value: 'fr'},
         {name: 'Galician', value: 'gl'},
         {name: 'German', value: 'de'},
+		{name: 'Greek', value: 'el'},
```
Comment (deepu105) [inline generators/generator-base.js]: formatting is wrong

File: test/test-languages.js [hunk 2]
```diff
@@ -19,6 +19,7 @@ describe('JHipster generator languages', function () {
         {name: 'Dutch', value: 'nl'},
         {name: 'Galician', value: 'gl'},
         {name: 'German', value: 'de'},
+		{name: 'Greek', value: 'el'},
```
Comment (deepu105) [inline test/test-languages.js]: formatting indent is wrong

**Comment by jdubois** [comment]
Thanks but don't forget to update also the website, if it's not documented people will not know it's there

**Comment by andidev** [comment]
Could you also translate the entity? You can find entity translations over here :
https://github.com/jhipster/generator-jhipster/tree/master/generators/entity/templates/src/main/webapp/i18n

**Pull request merged**